pauthabielf64/pauthabielf64.rst

```diff
@@ -542,167 +594,360 @@ by the run-time environment. There are a number of possible encoding
 schemas, each with its own trade-off. We are seeking feedback from
 platform owners about the most convenient form.
 
-Options for encoding the signing schema
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-To create a signed pointer the run-time system needs to know the
-signing schema to use for the pointer. The object producer and static
-linker will need to communicate this via metadata; including at least:
+Encoding the signing schema
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the descriptions below the ``place`` is the operation ``P`` in
+AAELF64_ relocation descriptions. It is derived from the r_offset
+field of the relocation.
+
+The top 32-bits of the contents of the place is used to encode the
+signing schema for both static and dynamic relocations. This permits
+platforms using relocation compression or SHT_REL dynamic relocations
+to encode relocation addends in the bottom 32-bits. Given that the
+maximum size of link-units using the small code-model is 4 gigabytes
+this should be sufficient.
+
+.. class:: pauthabiencoding
```

Comment:
if you're not going to use the class, you might as well delete it. also it's not adhering to the unofficial scheme: nameofdoc-description

Reply:
I've deleted all of the .. class:: as I don't think I use any of them.